include/solvers/mf_navier_stokes.h

```diff
@@ -30,13 +37,15 @@ using namespace dealii;
  * @tparam dim An integer that denotes the dimension of the space in which
  * the flow is solved.
  */
-template <int dim>
+template <int dim, typename OperatorType>
```

Comment:
I would say that you don't need to make `OperatorType` a template parameter. Could you post the compilation error which forced to used this.